src/webaudio.js

```diff
@@ -10,15 +10,6 @@ WaveSurfer.WebAudio = {
         return !!(window.AudioContext || window.webkitAudioContext);
     },
 
-    getAudioContext: function () {
```

Comment:
Please don't remove the function itself, it's used here: https://github.com/katspaugh/wavesurfer.js/blob/33e01a6be465d328d0712a7836091fd6efc6d869/plugin/wavesurfer.microphone.js